packages/notifi-dapp-example/src/app/notifi/ftu/page.tsx

```diff
@@ -0,0 +1,39 @@
+'use client';
+
+import { ConfigAlertModal } from '@/components/ConfigAlertModal';
+import { ConfigDestinationModal } from '@/components/ConfigDestinationModal';
+import { useNotifiCardContext } from '@/context/notifi/NotifiCardContext';
+import {
+  FtuStage,
+  useNotifiClientContext,
+  useNotifiSubscriptionContext,
+} from '@notifi-network/notifi-react-card';
+import { useEffect } from 'react';
+
+export default function NotifiFTU() {
+  const {
+    frontendClientStatus: { isInitialized },
+  } = useNotifiClientContext();
+
+  if (!isInitialized) return null;
+
+  const { ftuStage, syncFtuStage } = useNotifiSubscriptionContext();
+
+  const { cardConfig } = useNotifiCardContext();
```

Comment:
Shall we handle the case ftuStage is `ftuStage.Done` ??
It seems like if it is `Done`, the page will hang on still

Reply:
if the  ftuStage === ftuStage.Done, we'll show the dashboard page, I have added the the logic in useNotifiRouter
    if (frontendClientStatus.isAuthenticated && ftuStage === FtuStage.Done) {
      handleRoute('/notifi/dashboard');
      return;
    }